### ISSUE TITLE
Outlook automatic sideloading doesn't work

### ISSUE THREAD

**Issue by foofaraw** [OfficeDev/office-js-docs-pr#2505]
When running `npm start` on a freshly scaffolded Outlook project:

```
Sideloading the Office Add-in...
Error: Unable to start debugging.
office-addin-debugging: Unable to sideload the Office Add-in.
Error: Sideload is not supported.
```

According to StackOverflow posts linked below, Outlook sideloading is not supported. Is this page simply wrong?

https://stackoverflow.com/questions/57743701/unable-to-sideload-the-office-add-in-when-running-npm-start
https://stackoverflow.com/questions/58501403/debugging-outlook-add-in-desktop

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 3f364ae0-b7dd-edb3-b51b-8f6d0d99afbe
* Version Independent ID: 7dd1ce8f-16e7-88f8-f291-4afbe76baba7
* Content: [Sideload Outlook add-ins for testing - Office Add-ins](https://docs.microsoft.com/en-us/office/dev/add-ins/outlook/sideload-outlook-add-ins-for-testing)
* Content Source: [docs/outlook/sideload-outlook-add-ins-for-testing.md](https://github.com/OfficeDev/office-js-docs-pr/blob/master/docs/outlook/sideload-outlook-add-ins-for-testing.md)
* Product: **outlook**
* Technology: **add-ins**
* GitHub Login: @o365devx
* Microsoft Alias: **o365devx**

**Comment by AlexJerabek** [comment]
Thanks for reporting this @foofaraw. 

@ElizabethSamuel-MSFT, can you please investigate?

**Comment by ElizabethSamuel-MSFT** [comment]
@mscharlock Can you take a look? Thx.

**Comment by ElizabethSamuel-MSFT** [comment]
@foofaraw While we dig into this, can you try the following?

1. Run `npm i office-addin-debugging` then try `run start`
2. If that doesn't help, then do `npm run dev-server` and [manually sideload the add-in](https://docs.microsoft.com/office/dev/add-ins/outlook/sideload-outlook-add-ins-for-testing#sideload-manually)

Let us know if any of this helped.

Thx.

**Comment by foofaraw** [comment]
@ElizabethSamuel-MSFT Installing `office-addin-debugging` package doesn't help. Sideloading manually works fine.

**Comment by ElizabethSamuel-MSFT** [comment]
@foofaraw Thanks for letting us know. We'll investigate and make any necessary updates.

**Comment by ElizabethSamuel-MSFT** [comment]
@TCourtneyOwen Can you take a look? Thx.

**Comment by mscharlock** [comment]
Hi there - just an update, we do have engineers working on fixing this but we are able to repro your issue and it is indeed not working right now. Apologies for the frustration and delay in getting back to you. I'll keep you updated as things change. Thanks!

**Comment by millerds** [comment]
I was not able to repro this.  I followed the getting started guide at https://docs.microsoft.com/en-us/office/dev/add-ins/quickstarts/outlook-quickstart?tabs=yeomangenerator and the add-in side loaded just fine.

Note that we recently made some changes to fix and issue that was related to updates in npm v7 that were giving us trouble with the dev server starting.  It should be working for both v6 and v7 now.
